client/scss/layouts/_login.scss

```
@@ -82,6 +82,10 @@
     font-weight: theme('fontWeight.bold');
   }
 
+  .w-field__input {
+    padding:0rem;
```

Please use non-unt values for zero (see the linting error on the CI). Plus it may be nice to add a comment for this change. 
```suggestion
    padding: 0; // reset padding used within the admin
```